app/cypress/support/commands.js

```diff
@@ -158,10 +157,10 @@ Cypress.Commands.add('createProduct', (body) => (
 ))
 
 Cypress.Commands.add('connectToClient', (options = {}) => {
-    const client = new StreamrClient(getClientConfig(Object.assign({
-        restUrl: 'http://localhost/api/v1',
+    const client = new StreamrClient(getClientConfig({
         url: 'ws://localhost/api/v1/ws',
```

Comment:
I don't think the `url` config option exists anymore

Reply:
I ignored cypress specs on purpose, more or less. They need a general overhaul. Good catch tho.